docs/challenge/submission.md

```diff
@@ -0,0 +1,71 @@
+# Submission
+
+- **Submission Portal**: [Google Form](https://forms.gle/54tVqi5zs3ANGutn7)
+
+- **Leaderboard**: [here](./leaderboard.md)
+
+
+## **Submission details**
+
+After running the eval script, there will be two output files: an json file containing the metric results, and a mp4 video recording of the rollout trajectory. Here is a sample output json file for one episode of evaluation:
```

Comment:
Unclear what eval script they are running, how did we even get here, what does this have to do with submission?

Actually I think it's better if I make a pass at this file as an outsider, you are too familiar with this stuff to be able to write a fresh entry. Let's merge it as is and I'll take a shot at it tomorrow.